sample_app/app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.authenticate(params[:session][:password])
+      log_in user
+      redirect_to user
+    else
+      flash.now[:danger] = 'Invalid email/password combination' # 本当は正しくない
+      render 'new'
```

Comment:
```suggestion
      render :new
```

こういう書き方もできた気がします。
renderについては、gajoenではシンボル(コロンがつくもの)の書き方が多いですね。